src/pretix/static/pretixbase/scss/_dialogs.scss

```diff
@@ -0,0 +1,172 @@
+/* Modal dialogs using HTML5 dialog tags for accessibility */
+dialog.modal-card {
+  border: none;
+  width: 80%;
+  max-width: 43em;
+  padding: 0;
+  box-shadow: 0 7px 14px 0 rgba(78, 50, 92, 0.1),0 3px 6px 0 rgba(0,0,0,.07);
+  background: white;
+  border-radius: $border-radius-large;
+
+  opacity: 0;
+  transition: opacity .5s allow-discrete;
+
+  .modal-card-inner {
+    display: flex;
+    flex-direction: column;
+    align-content: stretch;
+  }
+  .modal-card-icon {
+    background: $brand-primary;
+    font-size: 2em;
+    color: white;
+    text-align: center;
+    padding: 3px;
+    .rotating {
+        -webkit-animation: fa-spin 8s infinite linear;
+        animation: fa-spin 8s infinite linear;
+    }
+  }
+  .modal-card-content {
+    padding: 1.5em;
+  }
+  .modal-card-content>*:last-child {
+    margin-bottom: 0;
+  }
+  .modal-card-content>*:first-child {
+    margin-top: 0;
+  }
+
+  .modal-card-confirm {
+    margin-top: 2em;
+    display: flex;
+    justify-content: flex-end;
+    gap: 1em;
+    align-items: center;
+  }
+  .modal-card-confirm-spread {
+    justify-content: space-between;
+  }
+}
+dialog::backdrop {
+  background-color: rgba(255, 255, 255, .5);
+  opacity: 0;
+  transition: opacity .5s allow-discrete;
+  backdrop-filter: blur(2px);
+}
+dialog[open], dialog[open]::backdrop {
+  opacity: 1;
+}
+@starting-style {
+  dialog[open], dialog[open]::backdrop  {
+    opacity: 0;
+  }
+}
+
+@media screen and (min-width: $screen-sm-min) {
+  dialog.modal-card:has(.modal-card-icon) .modal-card-inner {
+    flex-direction: row;
+    .modal-card-content {
+      padding: 2em;
+    }
+    .modal-card-icon {
+      font-size: 4em;
+      padding: 6px 16px;
+    }
+  }
+}
+
+.shake-once {
+  animation: shake .2s;
+  transform: translate3d(0, 0, 0);
+  backface-visibility: hidden;
+}
+
+@keyframes shake {
+  0% { transform: skewX(0deg); }
+  20% { transform: skewX(-5deg); }
+  40% { transform: skewX(5deg); }
+  60% { transform: skewX(-5deg); }
+  80% { transform: skewX(5deg); }
+  100% { transform: skewX(0deg); }
+}
+
+
+
```

Comment:
Do we want to keep the animation in there? AFAIK we do not use it anywhere as we do allow to cancel any dialogs. Or maybe we should add it to the loading-modal if someone tries to cancel with escape?